includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -4125,6 +4135,43 @@ public function get_validated_option( string $field_key, $empty_value = null ) {
+			if ( ! array_key_exists( $stripe_id, WC_Stripe_Payment_Tokens::UPE_REUSABLE_GATEWAYS_BY_PAYMENT_METHOD ) ) {
+				continue;
+			}
+
+			$gateway_id = WC_Stripe_Payment_Tokens::UPE_REUSABLE_GATEWAYS_BY_PAYMENT_METHOD[ $stripe_id ];
+
+			// parent::get_tokens() already loaded the main stripe gateway.
+			if ( $this->id === $gateway_id ) {
+				continue;
+			}
+
+			$method_tokens = WC_Payment_Tokens::get_customer_tokens( get_current_user_id(), $gateway_id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Payment_Tokens' in method 'get_tokens'. (undefined)

(StaticAccess)